### PR TITLE
SC38 - WIP

### PR DESCRIPTION
**Pull request by dzacharo**

### CODE REVIEW

File: docs/BR.md [hunk 1]
```diff
@@ -1468,7 +1468,9 @@ The CA SHALL verify that the Delegated Third Party's personnel involved in the i
 
 ### 5.4.1 Types of events recorded
 
-The CA and each Delegated Third Party SHALL record details of the actions taken to process a certificate request and to issue a Certificate, including all information generated and documentation received in connection with the certificate request; the time and date; and the personnel involved. The CA SHALL make these records available to its Qualified Auditor as proof of the CA’s compliance with these Requirements.
+The CA and each Delegated Third Party SHALL record events related to the security of Certificate Systems, Certificate Management Systems, Root CA Systems and Delegated Third Party Systems, especially events and details of the actions taken to process a certificate request and to issue a Certificate.
```
Comment (clintwilson) [inline docs/BR.md:1471]: Should we include a reference or pointer to the definitions of these in the NCSSRs?

File: docs/BR.md [hunk 2]
```diff
@@ -1534,9 +1540,36 @@ Additionally, the CA's security program MUST include an annual Risk Assessment t
...
+The CA and each Delegated Third Party SHALL retain, for at least two (2) years:
+
+1. CA certificate and key lifecycle management event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (1)) after the later occurrence of:
+    1. the destruction of the CA Private Key; or
+    2. the revocation or expiration of the final CA Certificate in that set of Certificates that have an X.509v3 `basicConstraints` extension with the `cA` field set to true and which share a common Public Key corresponding to the CA Private Key;
+2. Subscriber Certificate lifecycle management event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (2)) after the expiration of the Subscriber Certificate;
+3. Any security event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (3)) after the event occurred; 
+4. All archive records and documentation relating to the verification, issuance, and revocation of certificate requests and Certificates; and
```
Comment (clintwilson) [inline docs/BR.md]: Should this be "All archived records...." for consistency with bullet 5?

File: docs/BR.md (hunk 1 above)
Comment (dzacharo) [reply docs/BR.md:1471]: > 
> 
> Should we include a reference or pointer to the definitions of these in the NCSSRs?

Yes, this makes sense.

File: docs/BR.md (hunk 2 above)
Comment (dzacharo) [reply docs/BR.md]: fixed

File: docs/BR.md (hunk 1 above)
Comment (dzacharo) [reply docs/BR.md:1471]: Should we include a "catch all" phrase in section 1.6.1 pointing to definitions in the NCSSRs and possibly the EV Guidelines, or should we add specific references in section 5?

Comment (clintwilson) [reply docs/BR.md:1471]: I had been leaning towards adding them in section 5, but after giving it further thought I think 1.6.1 makes more sense as that seems an absence that is worth addressing regardless and may make it easier to both use and improve these NCSSRs-defined terms within the BRs in the future.

**Comment by clintwilson** [comment]
I've incorporated (I believe) all the feedback into a PR I plan to submit as a ballot (https://github.com/cabforum/servercert/compare/main...clintwilson:63dc6210e728349bb4602e4ede051efed593a91c).
I think the only thing I really ended up dropping was the illustrative examples. I do like, and am supportive, of putting more illustrative examples into the BRs, but this would be the first example of doing so and I'd like to ensure discussion focuses on the substantive changes rather than the introduction of illustrative examples.
Let me know if I've missed or misunderstood any of your other feedback and suggestions from this PR, as well as if you feel the illustrative examples are a vital part of the proposed changes in my PR.

**Comment by dzacharo** [comment]
I'm still in favor of at least the note in 5.5.2 and even 5.4.3 which has been discussed in various calls.

> **Note:** While these Requirements set the minimum retention period, the CA may choose a greater value as more appropriate in order to be able to investigate possible security or other types of incidents that will require retrospection and examination of past archive records.

I'm also in favor of illustrative controls for what "records" a CA must archive, following the RFC 3647 distinction between [audit logging](https://datatracker.ietf.org/doc/html/rfc3647#section-4.5.4) and [records archival](https://datatracker.ietf.org/doc/html/rfc3647#section-4.5.5). I can't really think of a negative outcome by including such guidance. We DO add substantive changes with this ballot but with the examples we also provide guidance for records archival that CAs and auditors can use.

**Pull request closed without merging**